charts/supaglue/templates/_helpers.tpl

```diff
@@ -137,6 +171,29 @@ and we want to make sure that the component is included in the name.
 {{- end -}}
 {{- end -}}
 
+{{- define "supaglue.api.managedDatabaseUrl" -}}
+{{- $databaseUrl := include "supaglue.managedDatabaseUrl" . -}}
+{{- $connectionLimit := ((.Values.api.db).parameters).connectionLimit -}}
+{{- $poolTimeout := ((.Values.api.db).parameters).poolTimeout -}}
+{{- $sslCert := empty ((.Values.api.db).parameters).sslCert | ternary nil (urlquery ((.Values.syncWorker.db).parameters).sslCert) -}}
+{{- $sslMode := ((.Values.api.db).parameters).sslMode -}}
+{{- $sslIdentity := ((.Values.api.db).parameters).sslIdentity -}}
+{{- $sslPassword := ((.Values.api.db).parameters).sslPassword -}}
+{{- $sslAccept := ((.Values.api.db).parameters).sslAccept -}}
+{{- $params := dict "connection_limit" $connectionLimit "pool_timeout" $poolTimeout "sslcert" $sslCert "sslmode" $sslMode "sslidentity" $sslIdentity "sslpassword" $sslPassword "sslaccept" $sslAccept -}}
```

Comment:
These values should come from the `managedDatabase.parameters` object, not the `api.db.parameters` one.